### ISSUE TITLE
Add aspectRation and optional alt text to lexicons/app/bsky/embed/external.json -> thumb

### ISSUE THREAD

**Issue by seboslaw** [bluesky-social/atproto#1890]
**Is your feature request related to a problem? Please describe.**

When requesting a post with an external embedding that has a thumb I have no way of knowing how large that thumb is. However, I need to know the aspect ration so that I can construct the cells in my table view. 
Also we're missing an alt text field for that thumb image. It could be optional and be extracted when the website is parsed.

**Describe the solution you'd like**

It would be cool, if lexicons/app/bsky/embed/external.json -> image also returned aspectRation and alt, just like lexicons/app/bsky/embed/images.json does already.
